### PR TITLE
Polish DB-API to support Python2 so can run on PAI

### PR DESCRIPTION
**Pull request by lhw362950217**
Related to #2781 

1. Enable absolute import in Python2 
1. Add param to super() function
1. Change exec to execute because exec in Python2 is a keyword
1. Add some driver specific API

### CODE REVIEW

File: python/runtime/dbapi/connection.py [hunk 1]
```diff
@@ -102,6 +102,11 @@ def __init__(self, conn_uri):
             if len(l) == 1:
                 self.params[k] = l[0]
 
+    def param(self, param_name, default_value=""):
```
Comment (sneaxiy) [inline python/runtime/dbapi/connection.py:105]: Seems that this method is not used?

File: python/runtime/dbapi/maxcompute.py [hunk 2]
```diff
@@ -81,18 +83,33 @@ class MaxComputeConnection(Connection):
...
+        self.params["database"] = proj
+        self.endpoint = endpoint
+        self._conn = ODPS(user, pwd, project=proj, endpoint=endpoint)
+
+    @staticmethod
+    def get_uri_parts(uri):
+        """Get username, password, endpoint, projectfrom given uri
+
+        Args:
+            uri: a vliad maxcompute connection uri
```
Comment (sneaxiy) [inline python/runtime/dbapi/maxcompute.py]: Typo `vliad`.

File: python/runtime/dbapi/connection.py (hunk 1 above)
Comment (lhw362950217) [reply python/runtime/dbapi/connection.py:105]: This is a tool api, we will need it in our submitter code.

**Review by sneaxiy — DISMISSED** [review]

File: python/runtime/dbapi/hive_test.py [hunk 3]
```diff
@@ -51,15 +51,15 @@ def test_query(self):
 
     def test_exec(self):
         conn = HiveConnection(testing.get_datasource())
-        rs = conn.exec("create table test_exec(a int)")
+        rs = conn.execute("create table test_exec(a int)")
```
Comment (typhoonzero) [inline python/runtime/dbapi/hive_test.py:54]: Is the test running on the CI currently?

Comment (lhw362950217) [reply python/runtime/dbapi/hive_test.py:54]: Not really, I will add it.

**Review by typhoonzero — APPROVED** [review]

**Pull request merged**